packages/strapi-plugin-users-permissions/models/User.settings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "connection": "default",
+  "connection": "user",
```

Comment:
Uhhh @lauriejim I don't think you want this set by default

Reply:
Oups

Reply:
Fixed